ironfish/src/mining/poolShares.ts

```diff
@@ -173,7 +170,7 @@ export class MiningPoolShares {
         w.poolPayoutSuccess(
           payoutId,
           transaction.content.hash,
-          transactionReceives,
+          transactionOutputs,
           shareCounts.totalShares,
         ),
       )
```

Comment:
Mat is re-writing this right now, can you link him to this to make sure this won't impact him? He might be able to just do this on his branch more easily.

Reply:
Chatted with Mat and reverted changes to `poolShares`